devs/docs/es-backports.rst

```diff
@@ -124,18 +125,18 @@ should be crossed out as well.
 - [ ] d13c1f50581 Fix Overly Strict Assertion in BlobStoreRepository (#63061) (#63236)
 - [ ] b4a1199e871 Uniquely associate term with update task during election (#62212)
 - [ ] c9baadd19bf Fix to actually throttle indexing when throttling is activated (#61768)
-- [ ] ba5574935e2 Remove dependency of Geometry queries with mapped type names (#63077) (#63110)
+- [s] ba5574935e2 Remove dependency of Geometry queries with mapped type names (#63077) (#63110)
 - [ ] 8c6e197f510 Remove allocation id from engine (#62680)
-- [ ] e28750b001e Add parameter update and conflict tests to MapperTestCase (#62828) (#62902)
-- [ ] 862fab06d3a Share same existsQuery impl throughout mappers (#57607)
-- [ ] 5ca86d541c5 Move stored flag from TextSearchInfo to MappedFieldType (#62717) (#62770)
+- [s] e28750b001e Add parameter update and conflict tests to MapperTestCase (#62828) (#62902)
+- [s] 862fab06d3a Share same existsQuery impl throughout mappers (#57607)
+- [s] 5ca86d541c5 Move stored flag from TextSearchInfo to MappedFieldType (#62717) (#62770)
 - [ ] cb1dc5260fb Dedicated threadpool for system index writes (#62792)
-- [ ] 3f856d1c81a Prioritise recovery of system index shards (#62640)
-- [ ] a0df0fb074b Search - add case insensitive flag for "term" family of queries #61596 (#62661)
-- [ ] 0d5250c99b1 Add Trace Logging to File Restore (#62755) (#62761)
-- [ ] 13e28b85ff5 Speed up RepositoryData Serialization (#62684) (#62703)
-- [ ] 803f78ef055 Add field type for version strings (#59773) (#62692)
-- [ ] 9a77f41e554 Fix cluster health when closing (#61709)
+- [s] 3f856d1c81a Prioritise recovery of system index shards (#62640)
+- [s] a0df0fb074b Search - add case insensitive flag for "term" family of queries #61596 (#62661)
+- [x] 0d5250c99b1 Add Trace Logging to File Restore (#62755) (#62761)
+- [x] 13e28b85ff5 Speed up RepositoryData Serialization (#62684) (#62703)
+- [s] 803f78ef055 Add field type for version strings (#59773) (#62692)
```

Comment:
https://github.com/elastic/elasticsearch/commit/803f78ef05562557dabb92638e546333dab578d6 is more like feature rather than patch + xpack change so skipped it anyway